Makefile

```diff
@@ -432,16 +435,18 @@ clean:
 	find . -type f -name '*_linux_amd64' -perm +u+x -exec rm {} \;
 	go clean ./...
 
-clean-protos:
+clean-protos: ## Clean protobuf files.
 	rm -rf $(PROTO_GOS)
 
 # List all images building make targets.
 list-image-targets:
 	@echo $(UPTODATE_FILES) | tr " " "\n"
 
-clean-doc:
+clean-doc: ## Clean the config file documentation.
 	rm -f $(DOC_TEMPLATES:.template=.md)
 
+check-doc: ## Check the config file is up to date.
+	@./tools/check-doc.sh
```

Comment:
What's this?

Reply:
I submitted the wrong code, probably related to a mistake in my rebase process. Sorry for confusing.